docs/integrations/builtin/credentials/huggingface.md

```diff
@@ -3,6 +3,8 @@
 title: Hugging Face credentials
 description: Documentation for the Hugging Face credentials. Use these credentials to authenticate Hugging Face in n8n, a workflow automation platform.
 priority: medium
+contentType: [integration, reference]
+priority: medium
```

Comment:
Looks like this is already above as well.

Reply:
:+1: